packages/ui/nuxt.config.js

```diff
@@ -129,6 +131,65 @@ export default {
     'nuxt-i18n',
     '@nuxtjs/apollo',
     'nuxt-element-ui',
+    '@nuxtjs/feed',
+  ],
+
+  feed: [
+    {
+      path: '/feed.xml', // The route to your feed.
+      async create(feed) {
```

Comment:
Could we move this to a single file to have all the business logic related with rss outside of the config file?